### PR TITLE
3.5.1. OBB/AABB API documents

### PR DESCRIPTION
**Pull request by PatriceJiang**
Re: #

Changelog:
 *

### CODE REVIEW

File: cocos/core/geometry/aabb.ts [hunk 1]
```diff
@@ -59,14 +57,20 @@ export class AABB {
     /**
       * @en
       * create a new AABB
+      * @param px  The x coordinate of the origin.
+      * @param py  The y coordinate of the origin.
+      * @param pz  The z coordinate of the origin.
+      * @param hw  Half the width of the AABB.
+      * @param hh  Half the height of the AABB.
+      * @param hl  Half the length of the AABB.
```
Comment (minggo) [inline cocos/core/geometry/aabb.ts]: The tool doesn't support this style. Should add both English comment and Chinese comment in the same line.

**Comment by GengineJS** [comment]
函数参数可以这么写吧？
image

File: cocos/core/geometry/aabb.ts [hunk 2]
```diff
@@ -59,14 +57,20 @@ export class AABB {
     /**
       * @en
       * create a new AABB
+      * @param px  The x coordinate of the origin.
```
Comment (minggo) [inline cocos/core/geometry/aabb.ts]: the origin -> the origin of the AABB

**Comment by minggo** [comment]
> 函数参数可以这么写吧？ image

Yep, should do like this. And please use English in future.

File: cocos/core/geometry/aabb.ts (hunk 1 above)
Comment (PatriceJiang) [reply cocos/core/geometry/aabb.ts]: Fixed

File: cocos/core/geometry/aabb.ts (hunk 2 above)
Comment (PatriceJiang) [reply cocos/core/geometry/aabb.ts]: Fixed

File: cocos/core/geometry/aabb.ts [hunk 3]
```diff
@@ -254,18 +257,21 @@ export class AABB {
...
       */
     public transform (m: Mat4, pos: Vec3 | null, rot: Quat | null, scale: Vec3 | null, out: AABB) {
         Vec3.transformMat4(out.center, this.center, m);
         transform_extent_m4(out.halfExtents, this.halfExtents, m);
     }
 
     /**
+      * @en
+      * Clones the AABB
+      * @returns A copy of the object.
```
Comment (minggo) [inline cocos/core/geometry/aabb.ts]: returns should combine in one line

File: cocos/core/geometry/aabb.ts [hunk 4]
```diff
@@ -287,7 +295,7 @@ export class AABB {
     /**
       * @en AABB and point merge.
       * @zh AABB包围盒合并一个顶点。
-      * @param point - 某一个位置的顶点。
+      * @param point @zh - 某一个位置的顶点。 @en input A point in 3d space.
```
Comment (minggo) [inline cocos/core/geometry/aabb.ts]: What is `input`?

File: cocos/core/geometry/aabb.ts [hunk 5]
```diff
@@ -307,10 +315,10 @@ export class AABB {
     }
 
     /**
-        * @en AABB and points merge.
-        * @zh AABB包围盒合并一系列顶点。
-        * @param points - 某一个位置的顶点。
-        */
+      * @en AABB and points merge.
+      * @zh AABB包围盒合并一系列顶点。
+      * @param points @zh - 某一个位置的顶点。 @en points A list of points in 3d space.
```
Comment (minggo) [inline cocos/core/geometry/aabb.ts]: points is not needed

File: cocos/core/geometry/aabb.ts [hunk 6]
```diff
@@ -321,7 +329,7 @@ export class AABB {
     /**
       * @en AABB and frustum merge.
       * @zh Frustum 合并到 AABB。
-      * @param frustum 输入的 Frustum。
+      * @param frustum @zh 输入的 Frustum。 @en frustum The frustum object.
```
Comment (minggo) [inline cocos/core/geometry/aabb.ts]: frustum is not needed

File: cocos/core/geometry/obb.ts [hunk 7]
```diff
@@ -226,11 +231,11 @@ export class OBB {
...
-     * @param m 变换的矩阵。
-     * @param pos 变换的位置部分。
-     * @param rot 变换的旋转部分。
-     * @param scale 变换的缩放部分。
-     * @param out 变换的目标。
+     * @param m 变换的矩阵。 * @param m The transform matrix
+     * @param pos 变换的位置部分。 * @param pos 3d-vector translation.
+     * @param rot 变换的旋转部分。 * @param rot Quaternion rotation.
+     * @param scale 变换的缩放部分。 * @param scale 3d-vector scale.
+     * @param out 变换的目标。 * @param out The output OBB.
```
Comment (minggo) [inline cocos/core/geometry/obb.ts]: Wrong format

File: cocos/core/geometry/aabb.ts [hunk 8]
```diff
@@ -241,8 +244,8 @@ export class AABB {
       * Get the bounding points of this shape
       * @zh
       * 获取 AABB 的最小点和最大点。
-      * @param {Vec3} minPos 最小点。
-      * @param {Vec3} maxPos 最大点。
+      * @param minPos @zh 最小点。 @en minPos Minimum position of the axis-aligned 3d bounding box.
+      * @param maxPos @zh 最大点。 @en maxPos Maximum position of the axis-aligned 3d bounding box.
```
Comment (minggo) [inline cocos/core/geometry/aabb.ts]: minPos and maxPos are not needed after @en

File: cocos/core/geometry/aabb.ts [hunk 9]
```diff
@@ -254,18 +257,21 @@ export class AABB {
...
-      * @param m 变换的矩阵。
-      * @param pos 变换的位置部分。
-      * @param rot 变换的旋转部分。
-      * @param scale 变换的缩放部分。
-      * @param out 变换的目标。
+      * @param m @zh 变换的矩阵。 @en m The transform matrix.
+      * @param pos @zh 变换的位置部分。 @en pos 3d-vector translation.
+      * @param rot @zh 变换的旋转部分。 @en rot Quaternion rotation .
+      * @param scale @zh 变换的缩放部分。 @en scale 3d-vector scale.
+      * @param out @zh 变换的目标。 @en out The output AABB.
```
Comment (minggo) [inline cocos/core/geometry/aabb.ts]: The same error.

**Review by minggo — APPROVED** [review]

**Comment by pandamicro** [comment]
https://github.com/cocos/cocos-engine/issues/10840

**Pull request merged**